app/structuring_the_knight_personality.py

```diff
@@ -0,0 +1,20 @@
+from typing import List, Dict, Optional
+
+
+class Knight:
```

Comment:
Think of adding methods here that could be beneficial to this class in order not to use it as a data structure only.

Reply:
👍
![just_be_creative](https://github.com/mate-academy/py-knights-fighting/assets/108234997/c01bc052-1ee2-423f-b82e-0355174d2612)

Reply:
merge preparation_battle.py and knight_class.py?

then what is the meaning of the packages?